bindata/etcd/pod.yaml

```diff
@@ -336,3 +360,10 @@ ${COMPUTED_ENV_VARS}
     - hostPath:
         path: /var/log/etcd
       name: log-dir
+    - hostPath:
+        path: /etc/kubernetes
+      name: config-dir
+    - hostPath:
+        path: /var/backup/etcd
+        type: DirectoryOrCreate
+      name: backup-dir
```

Comment:
how did we end up with `/var/backup`?

Reply:
I just used a diff path than the one used by the `AutomatedBackup` work 

Also I found this hostPath in the staticPod iirc

Reply:
removed from https://github.com/openshift/cluster-etcd-operator/pull/1304

it is still on https://github.com/openshift/cluster-etcd-operator/pull/1306 til we agree